### DISCUSSION TITLE
To restart a dag from failed task instead from beginning in airflow/cloud composer

### DISCUSSION THREAD

**Discussion by jeevanjoyappa** [apache/airflow#30413] [Q&A]
We have created 1 dag in cloud composer which has 2 tasks . Only after task A completes task B runs . Consider an instance where Task A completed and task B started but due to some code issue task B failed . We fixed the root cause of task B and rerun the job . The job instead of starting from task B it is starting from Task A. How do we make sure on the restart the job starts from the failed task and not from beginning ?

**Comment by potiuk** [comment]
You have to clear task B from existing job, not re-run the job.
